src/conf_mode/vpn_openconnect.py

```diff
@@ -39,6 +39,34 @@
 def get_hash(password):
     return crypt(password, mksalt(METHOD_SHA512))
 
+
+def T2665_default_dict_cleanup(origin: dict, default_values: dict) -> dict:
```

Comment:
Should we really reference task numbers in function names?

Reply:
Changed to _default_dict_cleanup